pkg/reconciler/events/storage/storage.go

```diff
@@ -43,12 +43,13 @@ import (
 )
 
 const (
-	reconciledSuccessReason      = "CloudStorageSourceReconciled"
-	reconciledNotificationFailed = "NotificationReconcileFailed"
-	reconciledPubSubFailed       = "PubSubReconcileFailed"
+	resourceGroup = "cloudstoragesources.events.cloud.google.com"
```

Comment:
this made me wonder if we needed the finalizerName in the other PR. Seems that we don't anymore. 
If you remember, when you create the next PR for a new reconciler, try to remove that finalizerName from the CloudPubSubSource. Otherwise no big deal, we can clean those up later on...
No need to do anything on this PR

Reply:
will do, sorry I did not notice that was no longer needed.